doc/content/reference/root-certificates/_index.md

```diff
@@ -26,7 +26,6 @@ Download the complete certificate list from curl [here](https://curl.haxx.se/ca/
 This `.pem` file contains certificates used in standard {{% tts %}} deployments, and is small enough to fit on memory constrained devices such as Gateways. This list includes the following CA certificates:
 
 - ISRG Root X1
-- DST Root CA X3
```

Comment:
BTW: You don't seem to update the `ca.pem` file.

Reply:
Ah ok that's our file. I thought it was copied directly from https://curl.haxx.se/ca/cacert.pem where DST still exists. Fixed.